### ISSUE TITLE
Not working proxy on Big Sur

### ISSUE THREAD

**Issue by flyer2001** [ProxymanApp/Proxyman#699]
### Proxyman version? (Ex. Proxyman 1.4.3)
Proxyman 2.12.0 (212000)
### macOS Version? (Ex. mac 10.14)
11.0.1

### Steps to reproduce 
After update to Big Sur not working web proxy from physical device. Most likely this is due to the settings of the operating system, but suddenly someone solved this problem. 
[these instructions ](https://docs.proxyman.io/troubleshooting/my-ios-devices-couldnt-connect-to-proxyman-via-proxy)don't help.

**Comment by NghiaTranUIT** [comment]
@flyer2001 can you try on this beta build: https://download.proxyman.io/beta/Proxyman_2.12.0_Big_Sur_Arm64.dmg

It's fully supported Big Sur and Apple Silicon Chip.

**Comment by NghiaTranUIT** [comment]
One more thing, physical device means Android or iOS devices?

**Comment by flyer2001** [comment]
> @flyer2001 can you try on this beta build: https://download.proxyman.io/beta/Proxyman_2.12.0_Big_Sur_Arm64.dmg
> 
> It's fully supported Big Sur and Apple Silicon Chip.

i tried this, didn't help

> One more thing, physical device means Android or iOS devices?
iPhone 6, iOS 12.4.8. Everything worked on Catalina

**Comment by NghiaTranUIT** [comment]
If your iPhone doesn't work. It means the Proxy IP is incorrect.

1. Can you verify what the IP in your wifi is? 

- By holding Option Key and click on the Wifi icon

<img width="316" alt="Screen_Shot_2020-11-18_at_11_23_14" src="https://user-images.githubusercontent.com/5878421/99483394-ada17080-2990-11eb-9e05-b1796e950734.png">

2. Does it the same as the IP you see in Proxyman -> Certificate -> Install on iOS Guideline?

**Comment by NghiaTranUIT** [comment]
I've tested with this build on my iPhone XR, iOS 12 and it works fine.

**Comment by flyer2001** [comment]
> If your iPhone doesn't work. It means the Proxy IP is incorrect.
> 
> 1. Can you verify what the IP in your wifi is?
> 
> * By holding Option Key and click on the Wifi icon
> 
> <img alt="Screen_Shot_2020-11-18_at_11_23_14" width="316" src="https://user-images.githubusercontent.com/5878421/99483394-ada17080-2990-11eb-9e05-b1796e950734.png">
> 
> 1. Does it the same as the IP you see in Proxyman -> Certificate -> Install on iOS Guideline?

yes the same

My iPhone not open http://proxy.man/ssl

**Comment by NghiaTranUIT** [comment]
If the IP in your wifi is the same from Mac and Your iPhone, and you couldn’t access any website. It might be the Firewall.

Please help me to double check that you enable Incoming traffic from Firewall macOS ( you can open it in system Preference > Security > Firewall tab

**Comment by flyer2001** [comment]
![image](https://user-images.githubusercontent.com/7876709/99488129-2b5d8000-2981-11eb-9f02-a600fcac4382.png)
firewall is off

**Comment by NghiaTranUIT** [comment]
Can you try on different iphone? And try to forget the network on your iphone too

**Comment by NghiaTranUIT** [comment]
I’m not sure what in your computer blocks the incoming traffic from iOS devices, but I have tested with my friends, iPhone X, iOS 14, and it’s fine.

**Comment by NghiaTranUIT** [comment]
@flyer2001 can you try to open http://proxy.man on Safari from your mac (proxyman is opening) to see if you can access to proxyman local server or not.

If you can, it means the problem might from the wifi network in some point.

**Comment by flyer2001** [comment]
From mac the local proxyman server opens. Trying to find the reason in the Wi-Fi network

**Comment by NghiaTranUIT** [comment]
Follow up from https://github.com/ProxymanApp/Proxyman/issues/371#issuecomment-729532660. 
The problem was the Endpoint Security VPN app. Quit this app and everything is working 🙌

**Comment by migfabio** [comment]
I am experiencing issues with Big Sur 11.3.1 with the latest version of Proxyman (2.25.0).
As soon as I opened it, I can see many random requests (from different IP addresses) failed, and after that, I cannot navigate. Internet stops working. Closing the Proxyman can't restore the internet connection, and I have to restart my mac.

I don't know how I can help you here.

**Comment by NghiaTranUIT** [comment]
@migfabio Can you try to do "Factory Reset" in Help Menu -> Advanced -> Factory Reset to see if it can fix your problem?

From my experience, look like you're opening VPN apps or (kernel extension, which is installed by your security team). They might conflict with Proxyman and other web debugging proxy too. Please confirm if it's not your cases 🤔

**Comment by NghiaTranUIT** [comment]
If it doesn't resolve your problem, please try to use Charles Proxy or Fiddler, and see if you're able to see the Traffic. If you can see the traffic from other apps, but not Proxyman. Look like the root of the problem is from Proxyman

**Comment by migfabio** [comment]
Thanks @NghiaTranUIT for your prompt response. I'll double-check with my IT team and see if there is software that conflicts with Proxyman because Charles does not work.

**Comment by NghiaTranUIT** [comment]
Thanks @migfabio please let me know the result 👍

**Comment by migfabio** [comment]
@NghiaTranUIT, it was the antivirus, and it's a known issue from the vendor. Thanks for your support.

**Comment by Hadleytron** [comment]
I am also having this issue ~ When I had the anti virus enabled, each domain was coming through as an IP address, and turning it off enabled the domain names to come through, but it still did not allow me to connect. If I have the 'Override macOS proxy' enabled, I just seem to be offline. I cannot visit any browser page or stay connected to MS Teams, which was never an issue when I was on mac OSCatalina
Certificate has been always trusted, etc. 

I have no issues with my mobile device proxying

macOS: Big Sur 11.4

When 'Override macOS proxy' enabled, my connection seems to go offline and I cannot process any traffic - 999 for every response: The operation couldn’t be completed. (NIO.ChannelError error 0.) (code=0)
![image](https://user-images.githubusercontent.com/34171663/122115323-9705af80-cde1-11eb-86ad-fc1667dccad4.png)

Any help is appreciated!

**Comment by NghiaTranUIT** [comment]
Hey @Hadleytron, can you try to do Factory Reset in the Help menu -> Advanced -> Factory Reset?

I'm currently running Proxyman under Big Sur 11.4 without any issues.

Look like the Proxyman Certificate is not trusted anymore in the System Keychain, after you upgrade to Big Sur. Factory Reset or delete the current certificate (Preference -> More button -> Delete) and install the certificate again, it might fix it 👍

**Comment by Hadleytron** [comment]
Thanks @NghiaTranUIT ! I tried deleting and installing a new certificate, but that has also not helped. This is my work laptop so I am not permitted to do a factory reset~ are there any other settings I might be able to re-configure?

**Comment by NghiaTranUIT** [comment]
@Hadleytron Delete the certificate and reinstall it should work.

From your screenshot, I can see the HTTPS Request content, it means that the certificate is correct. The problem is that Proxyman could not connect to the destination server (Look at the Response time Column, it's 10 seconds, which mean the connection timeout)

I suppose that your work machine might be installed in any VPN app (It might be installed by your Security Team)? Some Proxyman users have been encountered this problem because Sophos Security Extension is installed in theirsmachine. 

Ref: https://docs.proxyman.io/troubleshooting/proxyman-does-not-work-with-vpn-apps

**Comment by Hadleytron** [comment]
Hey @NghiaTranUIT ! So we are using Sophos, and verified that we are NOT using VPN when connecting to Proxyman, and Sophos central endpoint is on 10.0.4 version. Could there be anything else disrupting my connectivity?

**Comment by NghiaTranUIT** [comment]
@Hadleytron Yes, the problem causes by Sophos. If you look at the Reference Doc, one of Proxyman user mention:
```
1. Sophos
Sophos doesn't work with all Web Debugging Proxy apps. It's a known issue from Sophos and there is no solution to fix it.
Please try to ask your Security Team to try to set the HTTP/HTTPS Proxy from Sophos.

Update: Sophos 10.0.4 (A/V Endpoint Protection) might work with Proxyman with a new network proxy extension, but it has a known issue that causes massive CPU Spikes. It's going to fix in the upcoming EAP.

Credit: @AddictiveColors
```

If you're using Sophos 10.0.4, you can use Network Proxy Extension and config a Proxy to 127.0.0.1 at port 9090 (where Proxyman is)

Sophos causes this problem to all web debugging proxy apps too, includes Proxyman, Charles Proxy, Fiddler,... and there is no reliable solution to make it works smoothly.
